packages/client/src/App.test.tsx

```diff
@@ -127,39 +127,3 @@ describe('when user has a valid token in local storage', () => {
     expect(loadLocations).toHaveBeenCalled()
   })
 })
-
-describe('it handles react errors', () => {
```

Comment:
**Improvement:** I changed the error boundaries so that when running unit tests, errors won't get swallowed but instead are console.logged and thrown as you would expect in a test run